book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- Please implement a method to add rentals, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Please fix this 👍 👍

Reply:
Method add_rentals added.